webpack.config.js

```diff
@@ -56,54 +49,54 @@ module.exports = function (webpackEnvOptions) {
     'update-actions': false,
   };
 
-  return [
-    {
-      devtool: production ? undefined : 'cheap-module-source-map',
+  return {
+    devtool: production ? undefined : 'cheap-module-source-map',
```

Comment:
We were returning a list of one item, a relic from when we had multiple configs. I flattened it.